test/Sentry.Log4Net.Tests/SentryAppenderTests.cs

```diff
@@ -125,6 +130,19 @@ public void Append_NullEvent_NoOp()
         sut.DoAppend(null as LoggingEvent);
     }
 
+    [Fact]
+    public void Append_BelowThreshold_NoOp()
+    {
+        var sut = _fixture.GetSut();
+        sut.Threshold = Level.Warn;
+        var evt = new LoggingEvent(new LoggingEventData
+        {
+            Level = Level.Info
+        });
+
+        sut.DoAppend(evt);
```

Comment:
I'm not too sure what this test is checking. The only thing it really seems to test is whether an exception is thrown by `DoAppend`, in which case maybe we should rename it `Append_BelowThreshold_DoesNotThrow`.

Reply:
I wasn't sure how to write a test that verifies that no event has been sent. But now after searching through other tests in the solution I changed the assertion to:
```cs
_fixture.Hub.DidNotReceiveWithAnyArgs().CaptureEvent(Arg.Any<SentryEvent>());
```
Also renamed the test to `Append_BelowThreshold_DoesNotSendEvent`. Is it OK now?

Reply:
Ah, that makes heaps more sense 😄 The docs for NSubstitute are great by the way - it's a super powerful mocking framework.